go.mod

```diff
@@ -1,49 +1,74 @@
 module github.com/kubearchive/kubearchive
 
-go 1.21.9
+go 1.22.0
```

Comment:
I think we should leave of the `.0` at the end here. This will let compile with the latest version of go 1.22 without having to change this value. That's what most go projects do